src/main/java/org/truffleruby/core/string/StringNodes.java

```diff
@@ -178,6 +178,32 @@ public DynamicObject fromRope(Rope rope) {
             return executeMake(rope, NotProvided.INSTANCE, NotProvided.INSTANCE);
         }
 
+        public DynamicObject fromBuilder(RopeBuilder builder, CodeRange codeRange) {
+            return executeMake(builder.getBytes(), builder.getEncoding(), codeRange);
+        }
```

Comment:
This seems never used, should we keep it?

Reply:
I want to so if someone else is looking to make a string from a builder, they don't go down the wrong path. I'm working towards getting rid of RopeBuilder entirely and will clean this up at that time.

Reply:
Sounds good.